textproc/chordii/Portfile

```diff
@@ -0,0 +1,52 @@
+PortSystem          1.0
+
+name                chordii
+version             4.5.1
+revision            0
+categories          textproc
+maintainers         breun.nl:nils openmaintainer
+platforms           darwin
+license             GPL-2
+description         Chordii takes input files containing the lyrics and chords of one or more songs and produces a print (PostScript) version of these songs.
+long_description    Chordii, pronounced chord-ee-ee, is a utility that was \
+                    first created by lazy guitarists who got tired of turning \
+                    pages in the middle of the songs they liked. \
+                    \
+                    Chordii takes one or more input files containing the \
+                    lyrics and chords of one or more songs and produces a \
+                    print (PostScript) version of these songs. \
+                    \
+                    The output has the following characteristics: \
+                    * titles and sub-titles have been centered, \
+                    * the lyrics appear in the font and size of your choice, \
+                    * all chords names appear right above the right lyrics, \
+                    * all chords used in a song appear as grids at the bottom of the page. \
+                    \
+                    Optionally, you can also: \
+                    * generate an index of your songs, \
+                    * have the pages numbered, \
+                    * have chords transposed up or down, \
+                    * print in 2-up or 4-up modes (multiple pages per printed sheet) and, \
+                    * insert tablature and comments. \
+                    \
+                    You have a great many options on the final appearance of your songs.
+
+homepage            https://www.vromans.org/johan/projects/Chordii/
+master_sites        https://sourceforge.net/projects/chordii/files/chordii/4.5/
+distname            ${name}-${version}-mac
+worksrcdir          ${workpath}
+
+use_zip             yes
+checksums           rmd160  f2f836a7273d1301a24dc1311002e4752356f14f \
+                    sha256  16e3c63dc5a45112de3fe7d59f0a6a0fc8e66ec99d835e06a214d551f9414965
+
+use_configure       no
+
+build {}
+
+destroot {
+    # Copy over the binaries
+    foreach f { a2crd chordii } {
+        copy ${worksrcpath}/${f} ${destroot}${prefix}/bin
```

Comment:
Is there any reason why you would not build the software from source? MacPorts policy is to try to always build from source unless that cannot be done for some reason. The mac binaries are for PPC and i386 only. It would be much better to compile natively.